### DISCUSSION TITLE
[Cleanup] Too many branches.

### DISCUSSION THREAD

**Discussion by lolgear** [libgit2/libgit2#5958] [General]
This repository contains over 100 branches.
Could they be cleaned up or archived? ( I wish GitHub has "Archive" button for merged branches ).

**Comment by ethomson** [comment]
It does!  I cleaned up the branches that had been merged, but there remain several that have not been merged and may be stale that will take more investigation.

**Reply by lolgear** [reply]
I think that they are completely removed. 
I asked GitHub about this feature ( Archive but not Remove merged branches ).

But I have more common problem.
I forked repository several years ago and I don't know which remote branches are merged or removed.
It seems that I need to remove all remote branches that are not authored by me...
